### PR TITLE
Update ember-frost-date-picker to `^7.1.0`

### PR DESCRIPTION
**Pull request by dafortin**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

screen shot 2017-06-22 at 3 26 15 pm
screen shot 2017-06-22 at 3 26 23 pm
screen shot 2017-06-22 at 3 26 35 pm

# CHANGELOG
* **Updated** `ember-frost-date-picker@^7.1.0`

### CODE REVIEW

File: addon/components/inputs/when.js [hunk 1]
```diff
@@ -167,22 +166,24 @@ export default AbstractInput.extend({
...
-      // Needed because clicking in the date-time-picker changes the event object.
-      if (event.target.value === DATE_VALUE || event.target.value === firstButtonValue) {
-        this.set('selectedValue', event.target.value)
-      }
-
       // Set the bunsen model to the value of the selected button
       this.onChange(
         this.get('bunsenId'),
-        (event.target.value === firstButtonValue) ? firstButtonValue : this.get('storedDateTimeValue')
+        (value === firstButtonValue) ? firstButtonValue : (
```
Comment (sophypal) [inline addon/components/inputs/when.js]: This is harder to read then the previous implementation.

File: tests/integration/components/frost-bunsen-form/renderers/when-test.js [hunk 2]
```diff
@@ -83,7 +83,7 @@ describe('Integration: Component / frost-bunsen-form / renderer / when', functio
     it('renders as expected', function () {
       expectCollapsibleHandles(0)
       expectBunsenWhenRendererWithState('foo', {label: 'FooBar Baz'})
-      expectOnValidationState(ctx, {count: 2})
+      expectOnValidationState(ctx, {count: 4})
```
Comment (sophypal) [inline tests/integration/components/frost-bunsen-form/renderers/when-test.js]: I think this means that 2 more validations resulted from this change? How? I didn't see any change that would have contributed to this.

File: addon/components/inputs/when.js (hunk 1 above)
Comment (dafortin) [inline addon/components/inputs/when.js]: It's not exactly the same as previous implementation. I can try to make it clearer.

File: tests/integration/components/frost-bunsen-form/renderers/when-test.js (hunk 2 above)
Comment (dafortin) [inline tests/integration/components/frost-bunsen-form/renderers/when-test.js]: The implementation of the when function is a bit different and that's why we are getting more validation.

Comment (sophypal) [inline tests/integration/components/frost-bunsen-form/renderers/when-test.js]: Makes sense.

**Comment by sophypal** [comment]
👍

File: addon/components/inputs/datetime.js [hunk 3]
```diff
@@ -21,21 +25,30 @@ export default AbstractInput.extend({
...
+      return moment().format(DATE_TIME_FORMAT)
+    }
+    return value
+  },
+
   // == Functions ==============================================================
 
   parseValue (value) {
-    return value.format('YYYY-MM-DDTHH:mm:ssZ')
+    return moment(value).format(DATE_FORMAT)
```
Comment (sophypal) [inline addon/components/inputs/datetime.js]: Good catch!

**Pull request merged**